console/command.go

```diff
@@ -398,34 +399,44 @@ var getAddressesByAccountCmd = &cobra.Command{
 		getAddressesByAccount(account)
 	},
 }
-var importPriKeyCmd = &cobra.Command{
-	Use:   "importprivkey {priKey} {pripassword} {raw(hex), wif(Base58); Private Key format; default: raw}",
-	Short: "import priKey ",
-	Example: `
+
+func newImportPriKeyCmd() *cobra.Command {
+	var format string
+	importPriKeyCmd := &cobra.Command{
+		Use:   "importprivkey {priKey} {pripassword}",
+		Short: "import priKey ",
+		Example: `
 		importprivkey priKey pripassword ef235aacf90d9f4aadd8c92e4b2562e1d9eb97f0df9ba3b508258739cb013db2
-		importprivkey priKey pripassword 5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ wif
+		importprivkey priKey pripassword 5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ --format=wif
 		`,
-	Args: cobra.MinimumNArgs(2),
-	RunE: func(cmd *cobra.Command, args []string) error {
-		if err := OpenWallet(); err != nil {
-			return err
-		}
-		if err := UnLock(args[1]); err != nil {
-			return err
-		}
-		priv := args[0]
-		if len(args) >= 3 && args[2] == "wif" {
-			if decoded, _, err := qx.DecodeWIF(priv); err != nil {
+		Args: cobra.MinimumNArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := OpenWallet(); err != nil {
 				return err
-			} else {
-				priv = hex.EncodeToString(decoded)
 			}
-		}
+			if err := UnLock(args[1]); err != nil {
+				return err
+			}
+			priv := args[0]
+			if len(args) >= 3 && args[2] == "wif" {
```

Comment:
`args[2] == "wif"` is not correct
the code should like 
`format, _ := cmd.Flags().GetString("format")` 
cobra sould help you to tell the different between arguments and flags from your inputs paramers

Reply:
this has corrected in the following 27d8bc668c70da7da271ebd6ea1dee42d2c81212